src/lib/stateMappings/__tests__/singleIndex-test.ts

```diff
@@ -0,0 +1,133 @@
+import singleIndexStateMapping from '../singleIndex';
+
+describe('singleIndexStateMapping', () => {
+  describe('stateToRoute', () => {
```

Comment:
- What if the index name is not defined?
- What if the index name doesn't exist?

We should cover these cases.

Reply:
If the index isn’t defined an error will be thrown. That should probably be one we add ourselves. 

Test for that should indeed be given (undefined output now but might better be an empty object)